pkg/e2e/openshift/images.go

```diff
@@ -42,6 +78,11 @@ var _ = ginkgo.Describe(imageRegistryTestName, func() {
 		results, err := r.RetrieveResults()
```

Comment:
What are your thoughts on including both calling `ensurePassingXML` as well as checking that it's valid and passing as part of `r.RetrieveResults()` ?

I feel like that makes tests less redundant but I could see value in being explicit in each case.

Reply:
I think that's a good idea *if and only if* there is no usage of the helper that doesn't produce JUnit result data. If we always expect this, then we should absolutely do that.